### PR TITLE
Cleaning up misnamed variable in vtkPlusUsDevice

### PR DESCRIPTION
**Pull request by adamrankin**
Resolving redundant behaviour in some US device classes and the base class

Minor code formatting cleanup

### CODE REVIEW

File: src/PlusDataCollection/SonixVideo/vtkPlusSonixPortaVideoSource.cxx [hunk 1]
```diff
@@ -391,19 +393,7 @@ PlusStatus vtkPlusSonixPortaVideoSource::AddFrameToBuffer(void* param, int id, b
...
-    imageToTransducerTransformStr << this->CurrentPixelSpacingMm[0] << " 0 0 " << -1.0 * this->CurrentTransducerOriginPixels[0]*this->CurrentPixelSpacingMm[0];
-    imageToTransducerTransformStr << " 0 " << this->CurrentPixelSpacingMm[1] << " 0 " << -1.0 * this->CurrentTransducerOriginPixels[1]*this->CurrentPixelSpacingMm[1];
-    imageToTransducerTransformStr << " 0 0 " << zPixelSpacingMm << " 0";
-    imageToTransducerTransformStr << " 0 0 0 1";
-    customFields["ImageToTransducerTransform"] = imageToTransducerTransformStr.str();
-    customFields["ImageToTransducerTransformStatus"] = "OK";
-  }
-
-  PlusStatus status = aSource->AddItem(deviceDataPtr, aSource->GetInputImageOrientation(), frameSize, VTK_UNSIGNED_CHAR, 1, US_IMG_BRIGHTNESS, numberOfBytesToSkip, id, UNDEFINED_TIMESTAMP, UNDEFINED_TIMESTAMP, &customFields);
+  PlusStatus status = this->AddVideoItemToVideoSource(*aSource, deviceDataPtr, aSource->GetInputImageOrientation(), frameSize, VTK_UNSIGNED_CHAR, 1, US_IMG_BRIGHTNESS, numberOfBytesToSkip, id, UNDEFINED_TIMESTAMP, UNDEFINED_TIMESTAMP, &customFields);
```
Comment (lassoan) [inline src/PlusDataCollection/SonixVideo/vtkPlusSonixPortaVideoSource.cxx]: Why did you remove ImageToTransducerTransformName? It is an important feature.

Comment (adamrankin) [reply src/PlusDataCollection/SonixVideo/vtkPlusSonixPortaVideoSource.cxx]: It is redundant with the same variable in the base class.

Comment (lassoan) [reply src/PlusDataCollection/SonixVideo/vtkPlusSonixPortaVideoSource.cxx]: OK, I see. The content of this pull request looks good.

It would be better to have the commits a bit more focused, or at least have more accurate commit message ("Cleaning up misnamed variable in vtkPlusUsDevice" headline does not describe very well what was done).

Thanks for working on these.

**Comment by Sunderlandkyl** [comment]
There's currently an issue with the BkProFocusOem device.
http://perkdata.cs.queensu.ca/CDash/viewBuildError.php?buildid=101231

It's also used in the V4L2 device.

**Comment by adamrankin** [comment]
Fixed up BK device. V4L2 device was correct and used its own member container (just so happens to be named the same).

**Comment by adamrankin** [comment]
See https://github.com/PlusToolkit/PlusLib/commit/d16f775d0c5ce98e3b52e09910fd5df645d296a4
